packages/bento-design-system/src/Field/Config.ts

```diff
@@ -27,6 +27,9 @@ export type InputConfig = {
   radius: BentoSprinkles["borderRadius"];
   fontSize: BodyProps["size"];
   iconSize: IconProps["size"];
+  passwordShowIcon?: (props: IconProps) => Children;
+  passwordHideIcon?: (props: IconProps) => Children;
+  passwordIconSize?: IconProps["size"];
```

Comment:
the config entries should always be required (we process them to make them all optional when exposing this to the users)